.github/workflows/bleeding-edge.yml

```diff
@@ -13,14 +13,13 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
-        toolkit: ['null', 'pyside6', 'wx']
-        python-version: ['3.10']
-        exclude:
-          # No Wx wheels available for ubuntu or macos
-          - os: 'ubuntu-latest'
-            toolkit: 'wx'
-          - os: 'macos-latest'
+        toolkit: ['null', 'pyside6']
+        python-version: ['3.11']
+        include:
```

Comment:
Why are we including this for this version? Is it needed?

Reply:
This ensures that Wx code is still tested by this workflow

Reply:
@ankit-gautam23 This is a test suite which runs periodically and alerts Enthought devs if the tests fail due to upstream changes so we can take action.  We want some testing on WxPython, but it isn't easily installable for Python 3.11 yet.